internal/service/advancedclustertpf/resource_schema_test.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 func TestAdvancedCluster_ValidationErrors(t *testing.T) {
+	const (
+		projectID   = "111111111111111111111111"
```

Comment:
why wasn't this test failing before, do we also have projectID and clusterName at package level? same for next test

Reply:
Removed these from the package level to avoid accidental usage